README.md

```diff
@@ -1,5 +1,9 @@
 # MathSoc Website Transition
 
+The MathSoc web development team is rebuilding the MathSoc website to transition away from the current WordPress implementation.  We seek to rebuild it primarily using `pug` and `Express`, to allow for greater quality, maintainability, and customizability going forward.
+
```

Comment:
Thoughts on adding something here about scss and our cms (using json files for data instead of an sql server)?

Reply:
how does this sound?

Reply:
looks good